raitools/raitools/_managers/explainer_manager.py

```diff
@@ -126,13 +159,56 @@ def name(self):
         :return: The name of the explainer manager.
         :rtype: str
         """
-        return "explain"
+        return ManagerNames.EXPLAINER
 
     def save(self, path):
-        raise NotImplementedError(
-            "Save not implemented for ExplainerManager")
+        """Save the ExplainerManager to the given path.
+
+        :param path: The directory path to save the ExplainerManager to.
+        :type path: str
+        """
+        top_dir = Path(path)
+        # save the explanation
+        if self._explanation:
+            save_explanation(self._explanation,
+                             top_dir / ManagerNames.EXPLAINER)
+        meta = {IS_RUN: self._is_run,
+                IS_ADDED: self._is_added}
+        with open(Path(path) / META_JSON, 'w') as file:
+            json.dump(meta, file)
 
     @staticmethod
-    def load(path):
-        raise NotImplementedError(
-            "Load not implemented for ExplainerManager")
+    def load(path, rai_analyzer):
+        """Load the ExplainerManager from the given path.
+
+        :param path: The directory path to load the ExplainerManager from.
+        :type path: str
+        :param rai_analyzer: The loaded parent RAIAnalyzer.
+        :type rai_analyzer: RAIAnalyzer
+        """
+        # create the ExplainerManager without any properties using the __new__
+        # function, similar to pickle
+        inst = ExplainerManager.__new__(ExplainerManager)
+        top_dir = Path(path)
+        explanation_path = top_dir / ManagerNames.EXPLAINER
+        if explanation_path.exists():
+            explanation = load_explanation(explanation_path)
+            inst.__dict__['_' + ManagerNames.EXPLAINER] = explanation
```

Comment:
Could even just do `inst._explainer_manager=explanation`

What we really want here is something like C#'s `public explanation_manager { get; private set; }` but this is probably the best we can do.